Source/Syrup/Player/SyrupPlayerCharacter.h

```diff
@@ -4,6 +4,9 @@
 
 #include "CoreMinimal.h"
 #include "GameFramework/Character.h"
+#include "Syrup/Tiles/GridLibrary.h"
+#include "Syrup/Tiles/Plant.h"
```

Comment:
Plant include should be moved to .cpp and replaced with forward declaration as its functions are not used in this file.